File: kits/java/MyBot.java
import kit.*;
public class MyBot {
    public static void main(final String[] args) throws Exception {
        Agent agent = new Agent();

        while(true) {
            // wait for updates
            agent.update();
            // send some commands
            System.out.println("some command, another command");
            // end turn
            agent.endTurn();
            
        }
    }
}

File: kits/php/myBot.php
<?php
include(__DIR__ . "/kit.php");

$agent = new Agent();

// initialize the agent
$agent->initialize();

while(true) {
    // wait for engine updates
    $agent->update();
    // send some commands
    print "some command, another command\n";
    // end turn
    $agent->end_turn();
}